src/main/kotlin/org/ionproject/integration/builder/implementations/IselTimetableTeachersBuilder.kt

```diff
@@ -0,0 +1,259 @@
+package org.ionproject.integration.builder.implementations
+
+import com.squareup.moshi.Types
+import java.time.Duration
+import java.time.LocalTime
+import org.ionproject.integration.builder.exceptions.TimetableTeachersBuilderException
+import org.ionproject.integration.builder.interfaces.TimetableTeachersBuilder
+import org.ionproject.integration.model.internal.tabula.Cell
+import org.ionproject.integration.model.internal.tabula.Table
+import org.ionproject.integration.model.internal.timetable.Course
+import org.ionproject.integration.model.internal.timetable.CourseTeacher
+import org.ionproject.integration.model.internal.timetable.Event
+import org.ionproject.integration.model.internal.timetable.Faculty
+import org.ionproject.integration.model.internal.timetable.Label
+import org.ionproject.integration.model.internal.timetable.Programme
+import org.ionproject.integration.model.internal.timetable.School
+import org.ionproject.integration.model.internal.timetable.Teacher
+import org.ionproject.integration.model.internal.timetable.Timetable
+import org.ionproject.integration.model.internal.timetable.TimetableTeachers
+import org.ionproject.integration.model.internal.timetable.Weekdays
+import org.ionproject.integration.model.internal.timetable.isel.RawData
+import org.ionproject.integration.utils.JsonUtils
+import org.ionproject.integration.utils.RegexUtils
+import org.ionproject.integration.utils.Try
+
+class IselTimetableTeachersBuilder() : TimetableTeachersBuilder<RawData> {
+    val SCHOOL_REGEX = "\\A.*"
```

Comment:
These could declared as private `val` outside the class, right? We don't want them to be fields.